apps/wallet-ui/src/app/[ecosystem]/login/layout.tsx

```diff
@@ -0,0 +1,11 @@
+"use client";
+import { useRouter } from "next/navigation";
+import { useActiveAccount } from "thirdweb/react";
```

Comment:
unused?

Reply:
Yeah I'm surprised the linter didn't yell at me for these